usagov/usagov-2021
# Usagov 1914 aria role

## Jira Task


https://cm-jira.usa.gov/browse/USAGOV-

## Description

Removes an unsupported aria attribute on div around selections buttons.
Updates aria-role for the current page link in the pager to a supported value.

## Type of Changes

  - Add screenshot showing what it should look like

## Testing Instructions



### Change Requirements



### Validation Steps

- On search page, the div containing the buttons should not have an aria
- In search results, the element for the current page should have aria-current="page" in both English and Spanish pages

## Security Review




## Reviewer Reminders

- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions

Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
